packages/ui-components/src/components/ComboBox/ComboBox.stories.tsx

```diff
@@ -4,10 +4,12 @@
  */
 
 import React, { useState, useEffect } from "react"
-import PropTypes from "prop-types"
-import { ComboBox } from "./index.js"
-import { ComboBoxOption } from "../ComboBoxOption/index.js"
-import { PortalProvider } from "../../deprecated_js/PortalProvider/PortalProvider.component"
+import { ComboBox } from "./index"
+import { ComboBoxOption } from "../ComboBoxOption/index"
+import { PortalProvider } from "../PortalProvider/PortalProvider.component"
+import { ComboBoxProps } from "./ComboBox.component"
+
+type StoryFunction = () => React.ReactNode
 
 export default {
   title: "Forms/ComboBox/ComboBox",
```

Comment:
Please make [this change](https://github.com/cloudoperators/juno/pull/601/commits/f1fa2903825034370c3449a14afaf62ae6f69686) for all components in this PR and future ones 🙏
Storybook doesn't automatically pick up `ReactNode` as type and reverts to `any`.

Reply:
Done